pkg/controller/volume/cache/desired_state_of_world_test.go

```diff
@@ -987,10 +991,10 @@ func Test_GetVolumesToAttach_Positive_TwoNodesThreeVolumes(t *testing.T) {
 			generatedVolume2Name1,
 			generatedVolume2Name2)
 	}
-	pod3Name := types.UniquePodName("pod3-uid")
+	pod3Name := "pod3-uid"
 	volume3Name := api.UniqueVolumeName("volume3-name")
 	volume3Spec := controllervolumetesting.GetTestVolumeSpec(string(volume3Name), volume3Name)
-	generatedVolume3Name, podAddErr := dsw.AddPod(pod3Name, volume3Spec, node1Name)
+	generatedVolume3Name, podAddErr := dsw.AddPod(controllervolumetesting.NewPod(pod3Name, pod3Name), volume3Spec, node1Name)
 	if podAddErr != nil {
 		t.Fatalf(
 			"AddPod failed for pod %q. Expected: <no error> Actual: <%v>",
```

Comment:
Sorry, I meant the new `GetPodToAdd()` method not the `NewPod() test method. But only "if you have time".